concourse/import-exercises/script.deno.ts

```diff
@@ -0,0 +1,62 @@
+import { readCSVObjects } from 'https://deno.land/x/csv/mod.ts';
```

Comment:
I have an extension in vscode that is showing me a bunch of lint errors in this file according to `tslint.json`
I guess it doesn't really matter if this works

Reply:
yea i haven't found a linter that works for deno very well yet, the import format and the top level await anger my tslint